about.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Piano Concert</title>
+</head>
+
+<body>
+    <header>
+        <nav class="first-nav">
+            <ul class="first-ul">
+                <li>
+                    <a href="#"><img src="./images/fb.png" alt="facebookIcon"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="./images/twtr.png" alt="twitter"></a>
+                </li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+
+            </ul>
+        </nav>
+        <nav class="lower-nav">
+            <span class="menu" id="menu">
+        <ul class="hamburger">
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+    </span>
+            <a href="index.html"><img class="logo-second" id="second" src="images/piano logo.png" alt="logo"></a>
+            <ul class="mobile-list" id="mob-menu">
+                <li><i class="fas fa-times close-button" id="close-button"></i></li>
+                <li><a href="index.html" class="navigation-list" id="about-link">Home</a></li>
+                <li><a href="#" class="navigation-list">Program</a></li>
+                <li><a href="#" class="navigation-list">Join</a></li>
+                <li><a href="#" class="navigation-list">Sponsor</a></li>
+                <li><a href="#" class="button navigation-list">Summer Concert</a></li>
+            </ul>
+        </nav>
```

Comment:
- [OPTIONAL] Great work implementing the mobile navigation menu. I would suggest that you add a close button on the  menu so that it can be closed in case the user no longer wants to navigate to the next page.

![menu](https://github.com/Hsuwaddyhtat22/Microverse-first-capstone-project/assets/99656687/e5cc8a6d-a61d-4abe-8e06-ed551de02dab)